packages/ui/src/components/widget/index.tsx

```diff
@@ -877,7 +877,7 @@ const SwapWidget: FC<SwapWidgetProps> = ({
                   icon={faClock}
                   width={16}
                   style={{
-                    color: timeEstimate.time < 30 ? '#30A46C' : '#FFA01C'
+                    color: timeEstimate.time <= 30 ? '#30A46C' : '#FFA01C'
```

Comment:
Let's use the success color here and make a new color for warning? This will break theming